hypothesis-python/src/hypothesis/extra/array_api.py

```diff
@@ -778,10 +850,46 @@ def make_strategies_namespace(xp: Any) -> SimpleNamespace:
       >>> x
       Array([[-8,  6,  3],
              [-6,  4,  6]], dtype=int8)
-      >>> x.__array_namespace__() is xp
+      >>> x.__array_namespace__() is xp  TODO
       True
 
     """
+    check_argument(
+        api_version == Ellipsis
+        or (isinstance(api_version, str) and api_version in NOMINAL_VERSIONS),
+        f"{api_version=}, but api_version must be an ellipsis (...), or valid "
+        f"version string {RELEASED_VERSIONS}",
```

Comment:
"If the Array API standard version you want is not available, please ensure you're using the latest version of Hypothesis, then open an issue if one doesn't already exist."